shared/protocol/src/main/java/io/pravega/shared/protocol/netty/CommandEncoder.java

```diff
@@ -64,16 +65,13 @@
 @Slf4j
 public class CommandEncoder extends MessageToByteEncoder<Object> {
```

Comment:
My understanding is that the changes to this encoding are only about tracking, and I'd like a confirmation that this is not breaking compatibility with previous commits. If it does break, then we need to label the PR accordingly.

Reply:
yes.. these changes are only fix the append batch size tracker and reduces the block timeout events too. This PR does not change anything with encoding or data formatting.